packages/esm-appointments-app/src/metrics/metrics-card.component.tsx

```diff
@@ -29,7 +30,8 @@ const MetricsCard: React.FC<MetricsCardProps> = ({
   appointmentDate,
```

Comment:
Can you clean up the MetricsCard props? The only usage I see is [here](https://github.com/openmrs/openmrs-esm-patient-management/blob/20499f906983a27b1dac0f80a07382280fdbaaa4/packages/esm-appointments-app/src/metrics/appointments-metrics.component.tsx#L47-L66). Which means several of these props are not being used anywhere. You have deleted the only line that used `appointmentDate`, which means now you should remove `appointmentDate` from the interface. Looks like `view` isn't used by anyone anywhere either. Nor is `children`.

Reply:
For record-keeping: looks like `view` was added by @CynthiaKamau [here](https://github.com/openmrs/openmrs-esm-patient-management/pull/338) and partially removed by @donaldkibet [here](https://github.com/openmrs/openmrs-esm-patient-management/pull/652) and [here](https://github.com/openmrs/openmrs-esm-patient-management/pull/584). Dunno about the other props.